velox/exec/RowContainer.h

```diff
@@ -192,6 +192,27 @@ class RowContainer {
       RowColumn col,
       VectorPtr result);
 
+  /// Copies the values at 'col' into 'result' (starting at 'resultOffset')
+  /// for the 'numRows' rows pointed to by 'rows'. If a 'row' is null, sets
+  /// corresponding row in 'result' to null.
+  static void extractColumn(
+      const char* const* rows,
+      int32_t numRows,
+      RowColumn col,
+      const vector_size_t resultOffset,
+      VectorPtr result);
+
+  /// Copies the values from the array pointed to by 'rows' at 'col' into
+  /// 'result' (starting at 'resultOffset') for the rows at positions in
+  /// the 'rowNumbers' array. If a 'row' is null, sets
+  /// corresponding row in 'result' to null.
+  static void extractColumn(
+      const char* const* rows,
+      const BufferPtr& rowNumbers,
```

Comment:
Elsewhere we use folly::Range<char**>. Would it make sense to use the same here?

```
void eraseRows(folly::Range<char**> rows);
```